content/en/references/configuration.md

```diff
@@ -220,6 +220,7 @@ Please consult the [migration guide]({{< ref "user-guide/aws/lambda#migrating-to
 | `LAMBDA_LIMITS_CODE_SIZE_ZIPPED` | `52428800` (default) | The maximum zip file size in bytes for the [CreateFunction](https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html) operation. Raising this limit enables the creation of larger Lambda functions without the need to upload the code to an S3 deployment bucket. |
 | `LAMBDA_LIMITS_CREATE_FUNCTION_REQUEST_SIZE` | `70167211` (default) | The maximum HTTP request size in bytes for the [CreateFunction](https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html) operation. Raising this limit enables larger HTTP requests including zipped file size. |
 | `LAMBDA_LIMITS_MAX_FUNCTION_ENVVAR_SIZE_BYTES` | `4096` (default) | The maximum size of the environment variables that you can use to configure your function. |
+| `LAMBDA_PREBUILD_IMAGES` | `0` (default) |  Prebuild images before execution which increases the cold start time but reduces the time until the Lambda function is `ACTIVE`. |
```

Comment:
We should clearly label this as `(preview)` [feature](https://docs.localstack.cloud/references/changelog/#features-under-development) because it's not GA and not yet systematically tested.

/cc @dfangl